packages/@aws-cdk/aws-iam/lib/role.ts

```diff
@@ -148,6 +148,14 @@ export class Role extends Resource implements IRole {
   /**
    * Imports an external role by ARN.
    *
+   * If the imported Role ARN is a Token (such as a
+   * `CfnParameter.valueAsString` or a `Fn.importValue()`) *and* the referenced
+   * role has a `path` (like `arn:...:role/AdminRoles/Alice`), the
+   * `role.roleName` property will not resolve to the correct value. Instead it
+   * will resolve to the first path component. We unfortunately cannot express
+   * the correct calculation of the full path name as a CloudFormation
+   * expression.
```

Comment:
Should instead be a tsdoc comment on the `roleName` property of the `Import` class.

Reply:
Not really. No users will ever see it there, since the `Import` class is not public.